pyproject.toml

```diff
@@ -0,0 +1,66 @@
+[project]
+name = "opgee"
+version = "4.0.1a1"
+description = "Python 3 package implementing life cycle analysis of oil fields"
+authors = [
+  { name = "Michael Barlow", email = "mbarlow@rmi.org" },
+  { name = "Richard Plevin", email = "rich@plevin.com" },
+]
+dependencies = [
+  "chemicals>=1.2.0",
+  "dash>=2.18.1",
+  "dash-core-components>=2.0.0",
+  "dash-cytoscape>=1.0.2",
+  "dash-html-components>=2.0.0",
+  "dash-table>=5.0.0",
+  "dask>=2024.10.0",
+  "dask-jobqueue>=0.9.0",
+  "fluids>=1.0.26",
+  "lxml>=5.3.0",
+  "networkx>=3.4.2",
+  "numpy>=1.26.4",
+  "pandas>=2.2.3",
+  "pint>=0.24.3",
+  "pint-pandas==0.6.2",
+  "pydot>=3.0.2",
+  "python-dateutil>=2.9.0.post0",
+  "pyxsteam>=0.4.9",
+  "scipy>=1.14.1",
+  "semver>=3.0.2",
+  "sphinx>=8.1.3",
+  "sphinx-argparse>=0.5.2",
+  "sphinx-rtd-theme>=3.0.1",
+  "thermo>=0.3.0",
+  "thermosteam>=0.45.0",
+]
+requires-python = ">=3.11"
+readme = "README.md"
+license = { text = "MIT License" }
```

Comment:
@mbarlow12  If I understand correctly, this is implying that the entire repo is under the MIT license. If so, that's not the case, unfortunlately. But it looks like the pyproject spec supports 'license-file' as well as 'license'--so we can just point to the hybrid MIT-Stanford license that's in the repo? Also, that way if we're ever able to change the license, using a pointer means this will always refer to whatever is current. If that's the case, can we make that change, to use license-file rather than license?